app/src/main/AndroidManifest.xml

```diff
@@ -5,7 +5,7 @@
     <uses-permission android:name="android.permission.INTERNET" />
 
     <application
-        android:allowBackup="true"
+        android:allowBackup="false"
```

Comment:
it seemed to me like we shouldn't back this app's data up? I wouldn't want an apiToken for my notebook on the Google file servers.